arangod/Cluster/ActionBase.cpp

```diff
@@ -63,6 +63,22 @@ ActionBase::~ActionBase() {
   }
 }
 
+
+bool ActionBase::first() {
+  _actionStarted = std::chrono::system_clock::now();
+  return false;
+}
+
+void ActionBase::complete() {
```

Comment:
Redundant to MaintenanceWorker::nextState()'s management via setState() through its monitoring of _result.  Implies Action programmer must remember to call this or fail().